charts/fluentd/Chart.yaml

```diff
@@ -2,5 +2,13 @@ apiVersion: v2
 name: fluentd
 description: A Helm chart for Kubernetes
 # type: application
-version: 0.1.4
+version: 0.2.0
 appVersion: v1
+icon: https://www.fluentd.org/assets/img/miscellany/fluentd-logo_2x.png
+home: https://www.fluentd.org/
+sources:
+  - https://github.com/fluent/fluentd/
+  - https://github.com/fluent/fluentd-kubernetes-daemonset
+maintainers:
+  - name: edsiper
+    email: eduardo@treasure-data.com
```

Comment:
@edsiper  can you please take a look?

Reply:
Just an observation but the `appVersion` should probably work the same as the fluent-bit chart, a correct SemVer is always better here than an arbitrary string.

Reply:
Do you want to make the default image be built from the .Release.appVersion ?

Like:
image: "{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}"

The problem is that the specific image semVer compatible because of the "-debian-elasticsearch7-1.0"

I don't know specifically what is the difference between the images...

Reply:
You would use the `printf` command to template this out when using `appVersion` ([example](https://github.com/stevehipwell/helm-charts/blob/c6017998256a034c79835fee5d9d5bfb3af1cb90/charts/plantuml/templates/deployment.yaml#L54)).

Reply:
All set!